pkg/services/folder/folderimpl/folder_unifiedstorage_test.go

```diff
@@ -173,23 +177,19 @@ func TestIntegrationFolderServiceViaUnifiedStorage(t *testing.T) {
 		Host: folderApiServerMock.URL,
 	}
 
-	f := func(ctx context.Context) resource.ResourceClient {
-		return resourceClientMock{}
-	}
-
-	k8sHandler := &foldk8sHandler{
-		gvr:                    v0alpha1.FolderResourceInfo.GroupVersionResource(),
-		namespacer:             request.GetNamespaceMapper(cfg),
-		cfg:                    cfg,
-		restConfigProvider:     restCfgProvider.GetRestConfig,
-		recourceClientProvider: f,
-	}
-
 	userService := &usertest.FakeUserService{
 		ExpectedUser: &user.User{},
 	}
 
-	unifiedStore := ProvideUnifiedStore(k8sHandler, userService)
+	featuresArr := []any{
+		featuremgmt.FlagKubernetesFoldersServiceV2}
+	features := featuremgmt.WithFeatures(featuresArr...)
+
+	dashboardStore := dashboards.NewFakeDashboardStore(t)
+	_, err := unified.ProvideUnifiedStorageClient(cfg, features, db, tracing.InitializeTracerForTest(), prometheus.DefaultRegisterer, types.FixedAccessClient(true), &search.StandardDocumentBuilders{})
```

Comment:
why do we need to call this if we are not using the returned client?

Reply:
Good catch - you're right, we don't need this anymore.